Articles/Defining and Using Observables.md

```diff
@@ -0,0 +1,341 @@
+
+Whether creating observables manually -- a part of mapping process -- or automatically, care must be taken while populating the `name` and `value` fields.
+
+### Populating Observable Path References
+The `observable` object's `name` attribute is an attribute path reference. See [Appendix 1: Attribute Paths](#appendix-1-attribute-paths) for details.
+
+### Populating the Value Field
+The `observable` object's `value` field should be populated for all primitive types (strings, numbers, booleans), _and_ for arrays of primitive types (see next paragraph). The `value` field is specifically not meant to be used for observable objects, nor for for arrays of objects.
+
+For arrays of primitive types, one `observable` object should be created for each element of the array with the `value` field being set to the array element's value.
+
+#### All Observable Values Are Strings
+The `observable` object's `value` is defined as a type `string_t`. (Specifically notice that the type of `value` is not the problematic `json_t` type, which means any type.) 
+
+A primitive value that is not a string (either `string_t` or a subtype of `string_t`) must be converted to string.
+
+Suggested conversions of non-string values:
+* `integer_t` and `long_t`: base 10 string.
+* `float_t`: base 10 string using common standard library conversions, including exponential notation and "NaN".
+* `boolean_t`: the strings `"true"` and `"false"`.
+* Null should not be converted to string, but rather the encoding's equivalent of `null`. In other words, a null remains a null. In JSON encoded events, use `"value": null` and not `"value": "null"`.
+
+> [!NOTE]
+> About `null`, it's weird. Don't overthink it. OCSF does not have a null type. In practice this means OCSF does not distinguish between a field that has a `null` value and a missing field. For observables, when creating them for primitive fields (like strings and numbers), if the field's value is `null`, then you may either set the `observable` object's `value` to `null` or not set the `value` field -- the meaning of each is equivalent. (This is not true in general. For those of that remember the XML era, distinguishing `null` from missing was one of the consistently annoying edge cases you'd have to always keep in mind.)
+
+## Appendix 1: Attribute Paths
+Attribute paths occur in two places: in the `observable` object's `name` attribute as a path reference to a field in the event, and in class-specific attribute observable definitions. In both cases the paths are same. (Author's note: I believe this is the only use of JSON path-like capability in OCSF.)
```

Comment:
Grammar: "In both cases the paths are same." : should be 'the same'?

Is the first time you say "Author's note' vs. "Note"?  Is the the first time you use the first person?  Unless necessary, just using first person once in a document isn't recommended.